src/Model/File.php

```diff
@@ -49,6 +53,12 @@ protected function init(): void
         $this->addField('meta_image_width', ['type' => 'integer']);
         $this->addField('meta_image_height', ['type' => 'integer']);
 
+        $this->onHook(Model::HOOK_BEFORE_SAVE, function (self $m, bool $isUpdate) {
+            if ($isUpdate === false) {
```

Comment:
```suggestion
            if (!$isUpdate) {
```
as we work on native bool